pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -261,10 +262,54 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
 
+	poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", pool, getRadosNamespaceName(cephBlockPoolRadosNamespace))
```

Comment:
To follow up from discussion on checking version, you can query it from the `clusterInfo`, something like:
```go
clusterInfo.CephVersion.IsAtLeast(...)
```